ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -312,7 +312,9 @@ private void onChunkOperationComplete(GetChunk chunk) {
           // GetManager remove this operation.
           setOperationCompleted();
           List<StoreKey> chunkIds = e == null && compositeBlobInfo != null ? compositeBlobInfo.getKeys() : null;
-          operationResult = new GetBlobResultInternal(null, chunkIds);
+          operationResult = new GetBlobResultInternal(
+              options.getBlobOptions.getOperationType() == GetBlobOptions.OperationType.BlobChunkIds
```

Comment:
In `GetBlobOptionsInternal` there is a member called `getChunkIdsOnly`, we should probably get rid of that as well now that we have BlobChunkId as SubResource.

Reply:
I checked the undelete and delete also need this getChunkIdsOnly in GetBlobOptionsInternal, so I just keep using it. Let me know if I miss anything. thx!